data/data.js

```diff
@@ -1,73 +1,73 @@
+    "java": ["1995", "オブジェクト指向のプログラミング言語 プラットフォームに依存しない特性を持つ", "Java"],
+    "jl": ["2012", " 高性能な数値計算やデータ分析に使われる動的プログラミング言語", "Julia"],
+    "jst": ["1995", "クライアントサイドとサーバーサイドのウェブ開発に広く使われる言語", "JavaScript"],
+    "kn": ["2012", "日本製の教育用プログラミング言語 初心者でも学びやすいことを目指している", "Kuin"],
+    "kt": ["2011", " JVM上で動作する静的型付けのプログラミング言語 Javaと100%互換性がある", "Kotlin"],
+    "lol": ["2007", "ユーモラスなプログラミング言語 インターネットのミーム言語に基づいている", "LOLCODE"],
+    "lsp": ["2018", "/*unkown*/", "LSP"],
+    "lua": ["1993", "埋め込み可能なスクリプト言語 ゲーム開発や組み込みシステムによく使われる", "Lua"],
+    "ml": ["1973", "Meta Languageの略 静的型付けと型推論の機能を持つ関数型プログラミング言語", "ML"],
+    "moon": ["2011", "Luaに対する高級言語で、Luaにコンパイルされる", "MoonScript"],
+    "nako": ["2011", "日本語を基にしたプログラミング言語で、プログラミングの初学者や子供たちによる学習を目指している", "Nadesiko"],
+    "nim": ["2008", "高性能で表現力豊かなシステムプログラミング言語 Pythonライクなシンタックスを持ちつつも、C、C++、JavaScriptにトランスパイル可能", "Nim"],
+    "nscripter": ["1998", "日本製のビジュアルノベル作成用エンジン 専用のスクリプト言語を使用する", "NScripter"],
+    "p": ["1970", "教育用途に広く使われた手続き型のプログラミング言語 読みやすさと、良いプログラミング習慣を教えることを目指して設計された", "Pascal"],
+    "pde": ["2001", "Processing Development Environmentの略 ビジュアルアートとソフトウェアの教育を目的としたProcessing言語の統合開発環境", "PDE"],
+    "php": ["1995", "主にウェブ開発に用いられるサーバーサイドスクリプト言語 HTMLと組み合わせて使われることが多い", "PHP"],
+    "pl": ["1964", "論理プログラミング言語 人工知能の研究や自然言語処理に用いられる", "prolog"],
+    "ps1": ["2006", "Microsoftが開発したタスク自動化と構成管理のためのシェルスクリプト言語", "Powershell"],
+    "py": ["1991", "高レベルな汎用プログラミング言語 コードの可読性に重点を置いています", "Python"],
+    "raku": ["2000", "Perl 6とも呼ばれる高水準の汎用プログラミング言語 強力な正規表現と文法的な柔軟性が特徴", "Raku"],
+    "rb": ["1995", "簡潔さと可読性に焦点を当てたオブジェクト指向スクリプト言語", "Ruby"],
+    "rdr": ["2018", "/*unkown*/", "RDR"],
+    "re": ["2015", "OCamlに基づく静的型付けの関数型プログラミング言語 React.jsと組み合わせて使われることが多い。", "Reason"],
+    "res": ["2018", "JavaScriptの強力な型システムと、OCaml/Reasonのシンタックスを組み合わせた言語 JavaScriptへのコンパイルを目指して開発された", "Rescript"],
+    "rack": ["1995", "Lisp/Schemeファミリーに属する関数型プログラミング言語。教育や研究にも使用される", "Racket"],
+    "rs": ["2010", "安全性と並行性に焦点を当てたシステムプログラミング言語 C++の代替として期待されている", "Rust"],
+    "sb3": ["2003", "子供や初心者向けのビジュアルプログラミング言語 ブロックベースのインターフェースでコードを組み立てる", "Scratch"],
+    "scala": ["2003", "静的型付けのオブジェクト指向と関数型プログラミング言語 Javaとの互換性があり、JVM上で動作する", "Scala"],
+    "scm": ["1975", "Lispファミリーの最小主義的な関数型プログラミング言語 教育や研究で利用されることが多い", "Scheme"],
+    "svelte": ["2016", "ウェブアプリケーションのフロントエンド開発向けのフレームワーク コンパイラを使用して効率的なJavaScriptに変換される", "Svelte"],
+    "swift": ["2014", "Appleが開発したiOS、macOSアプリケーション開発用のプログラミング言語", "Swift"],
+    "tex": ["1978", "数学や物理学の文書作成に適したマークアップ言語 高品質なタイポグラフィが特徴", "TeX"],
+    "ts": ["2012", "JavaScriptの上位互換言語で、静的型付けを導入 JavaScriptにコンパイルされる", "TypeScript"],
+    "v": ["2019", "C/C++の代替として開発されたシンプルで高速なプログラミング言語", "V"],
+    "vb": ["1996", "Microsoftが開発したVisual Basicのスクリプト版 主にWindowsのスクリプトホストで実行される", "VBScript"],
+    "vim": ["1991", " Vimエディタの機能拡張やカスタマイズのためのスクリプト言語", "Vim script"],
+    "vue": ["2014", "ウェブアプリケーションのフロントエンド開発に使われるJavaScriptフレームワーク", "Vue.js"],
+    "wat": ["2018", "WebAssemblyのテキスト形式 低レベルのバイナリフォーマットで効率的な実行が可能", "wat"],
```

Comment:
それは wasm の説明であって wat はそれ以上の意味を持たないだろう

```suggestion
    "wat": ["2018", "WebAssemblyのテキスト形式 S式によって表現されている", "wat"],
```